src/main/python/ayab/engine/mode.py

```diff
@@ -79,6 +84,10 @@ def add_items(box):
         box.addItem(tr_("KnitMode", "Ribber: Circular"))
 
 
+if TYPE_CHECKING:
+    ModeTuple: TypeAlias = tuple[int,int,bool,bool]
+    ModeFuncType: TypeAlias = Callable[[Control,int],ModeTuple]
```

Comment:
limitation of the Alias implementation- if I don't fence this, the `Control` here creates a circular import.